.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
           E2E_TEST_ESS_POD: ${{ secrets.E2E_TEST_ESS_POD }}
       - run: npx prettier --check "{packages/*/src,packages/*/__tests__}/**"
       - run: npm audit --audit-level=moderate
+      - run: npm run lerna-audit -- --audit-level=moderate
```

Comment:
This looks like we should perhaps have a Lerna section in our 'Coding Guidelines' - i.e., I assume this should be our setup for all Lerna projects at Inrupt (i.e., I believe Arne is using Lerna too), and also to answer why 'moderate'...?

Reply:
To be honest, I copy-pasted the line above, so I'm not sure about `moderate`, but having general Lerna guidelines sounds like a good thing, even if it is just to capitalize knowledge.